XcodeServerSDK/Server Entities/SourceControlBlueprint.swift

```diff
@@ -27,6 +27,7 @@ public class SourceControlBlueprint : XcodeServerEntity {
     public let privateSSHKey: String?
     public let publicSSHKey: String?
     public let sshPassphrase: String?
+    public var certificateFingerprint: String?
```

Comment:
since its being set to `nil` on the init why not declare it with a default value?
```swift
public var certificateFingerprint: String? = nil
```

And only set it to something else when a value is passed?